modules/cms/formwidgets/components/partials/_component.htm

```diff
@@ -1,17 +1,17 @@
 <div class="layout-cell <?= e($component->componentCssClass) ?> <?= $component->isHidden ? 'hidden' : null ?>">
     <div
-        class="<?= 'oc-'.$component->pluginIcon ?> layout-relative"
+        class="layout-relative"
         <?php if ($component->inspectorEnabled): ?>data-inspectable<?php endif ?>
         data-inspector-title="<?= $name = e($this->getComponentName($component)) ?>"
         data-inspector-description="<?= $description = e($this->getComponentDescription($component)) ?>"
         data-inspector-config="<?= e($this->getComponentsPropertyConfig($component)) ?>"
         data-inspector-class="<?= get_class($component) ?>">
-            <span class="name"><?= $name ?></span>
+            <span class="name"><i class="<?= $component->pluginIcon ?>"></i> <?= $name ?></span>
```

Comment:
Doesn't this break it because now pluginIcon would be forced to include the `oc-` prefix?

Reply:
Good point, I will use `str_replace()`

Reply:
@LukeTowers Please review and let me know your thoughts: https://github.com/octobercms/october/pull/5429/commits/4652d2902301ef64190cef24461a4eaf1fcbec5e

Reply:
@LukeTowers ok please review this comment: https://github.com/octobercms/october/pull/5429/commits/fac649da1940cc79a6e80fee1f68aa8162221b90

Reply:
@LukeTowers Your comment doesn't apply to the cms's setup.

The incoming icon prefix into the plugin is: `icon-flag-checkered` and the reason why I opened this issue up in the first place was because October was injecting a hard-coded `oc-` into the string. See code line:

https://github.com/octobercms/october/blob/develop/modules/cms/formwidgets/components/partials/_component.htm#L3

If someone put this in their plugin's component `oc-icon-flag-checkered` then the output from October would be this:

```
oc-oc-icon-flag-checkered
```

Notice the double `oc-oc-`.

So before this issue was open, people could not add `oc-` to the icon name for their plugin component.

So there is no need to remove it, when people haven't been able to add it in the first place.